gax-httpjson/src/main/java/com/google/api/gax/httpjson/ProtoMessageResponseParser.java

```diff
@@ -79,7 +79,7 @@ public ResponseT parse(Reader httpContent, TypeRegistry registry) {
   /* {@inheritDoc} */
   @Override
   public String serialize(ResponseT response) {
-    return ProtoRestSerializer.create(defaultRegistry).toJson(response);
+    return ProtoRestSerializer.create(defaultRegistry).toJson(response, false);
```

Comment:
@vam-google I couldn't find any usage of this method, is this method only used by tests? It also feels strange to me that a response parser need to serialize an object to String.

Reply:
Yes, I believe it is used only by tests, but that also counts as "usage", right? I think this "serialize" interface thing predates my time on rest transport, so I just had to keep it there to not break things.

Reply:
Cool, I think passing `false` is appropriate then.

Reply:
However, we should probably add some comments to indicate this public method is only used in tests to set up things? Or we can make it better by using a different way to serialize object [here](https://github.com/googleapis/gax-java/blob/4f82b8d0a963295d947f2dab5151fc109205fc8c/gax-httpjson/src/test/java/com/google/api/gax/httpjson/testing/MockHttpService.java#L231), so that we can remove this test only method?